_books/ion-1-1/src/whats_new.md

```diff
@@ -224,23 +84,27 @@ Within a list or S-expression, the stream becomes additional child elements in t
 
 **E-expressions in S-expressions**
 ```ion
+// Encoding
 (a (:values 1 2 3) b (:none) c)
+
+// Evaluates to
 (a 1 2 3 b c)
 ```
 
+**E-expressions in structs**
+
 Within a struct at the field name position, the resulting stream must contain structs and each of the fields in those
 structs become fields in the enclosing struct (the value portion is not specified); at the value position, the resulting
-stream of values becomes fields with whatever field name corresponded before the E-expression (empty stream elides the
-field all together). In the following examples, let us define `(:make_struct c 5)` that evaluates to a single struct
+stream of values becomes values with whatever field name corresponded before the E-expression (empty stream elides the
```

Comment:
I think `fields` was a bit more descriptive here?

Reply:
Ah, I suppose that depends on whether you think of fields as having a value (as modeled in ion-rust) or values as having a field name (as modeled in ion-java). This is my ion-java preconceptions shining through. Happy to change it back.